asminc/pce.inc

```diff
@@ -0,0 +1,101 @@
+;
+; PCE definitions. By Groepaz/Hitmem.
+;
+
+; FIXME: optimize zeropage usage
+CURS_X          = $30
+CURS_Y          = $31
+SCREEN_PTR      = $32           ;2
+CHARCOLOR       = $34
+RVS             = $35
+BGCOLOR         = $36
+tickcount       = $37           ;4
```

Comment:
I recommend that you put those variables into an "`extzp.s`" file.  If you define them in an `EXTZP` segment, then you won't need to manage their addresses by hand.

(See the `libsrc/*/extzp.*` files.)